src/industry_cmd.cpp

```diff
@@ -1667,6 +1667,33 @@ static CommandCost CheckIfFarEnoughFromConflictingIndustry(TileIndex tile, int t
 	return CommandCost();
 }
 
+/**
+ * Check whether the tile is a mine.
+ * @param tile the tile to investigate.
+ * @return true if and only if the tile is a mine
+ */
+bool CheckIfIndustryIsAMine(TileIndex tile)
```

Comment:
i guess it would fit our naming scheme better if this was named `IsIndustry<Something>`

Reply:
Done! Thanks for the suggestion.